docs/src/maintain/overview.md

```diff
@@ -0,0 +1,44 @@
+---
+title: How ESLint is Maintained
+eleventyNavigation:
+    key: how eslint is maintained
+    parent: maintain eslint
+    title: How ESLint is Maintained
+    order: 1
+
+---
+
+This page explains the different roles and structures involved in maintaining ESLint.
+
+## The ESLint Team
+
+The ESLint team works together to develop and maintain ESLint. To learn more about the different roles on the ESLint team, refer to [Governance](../contribute/governance). To see the current team members, refer to [Team](https://eslint.org/team/).
+
+## Organization Structure
+
+ESLint is part of the [OpenJS Foundation](https://openjsf.org/), a nonprofit organization that supports open-source projects and communities in the JavaScript ecosystem.
+
+The OpenJS Foundation provides legal infrastructure for JavaScript projects like ESLint. It is the owner of the intellectual property related to ESLint, including copyrights and trademarks, and ensures the independence of the project. They are also a resource for ESLint if we need legal advice or representation.
+
+The OpenJS Foundation does not participate in the day-to-day functioning of ESLint.
+
+## Funding
+
+ESLint is funded through several sources, including:
+
+* [**Open Collective**](https://opencollective.com/eslint): A platform for financing open source projects.
+* [**GitHub Sponsors**](https://github.com/sponsors/eslint): A platform for funding open source projects associated with Github.
+* [**Tidelift**](https://tidelift.com/subscription/pkg/npm-eslint): A subscription service that lets enterprises manage and fund the open source projects that their organization uses.
+* [**Carbon Ads**](https://www.carbonads.net/open-source): Developer-centric advertising provider used on eslint.org.
```

Comment:
```suggestion
* [**Carbon Ads**](https://www.carbonads.net/open-source): Developer-centric advertising provider used on [eslint.org](https://eslint.org/).
```

Since this gets auto-linked to `http://eslint.org`, seems better to make it a link to `https://eslint.org` in the source.